salt/utils/cloud.py

```diff
@@ -2262,6 +2289,13 @@ def sftp_file(dest_path, contents=None, kwargs=None, local_file=None):
         ssh_args = [
             # Don't add new hosts to the host key database
```

Comment:
I also noticed the function `def root_cmd` has `ssh_args` as well. Does that function also need to be updated to allow a user to pass in these two new kwargs?